ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Expected Behavior
+
+## Current Behavior
+
+## Possible Solution (we welcome any logical suggestion)
+
+## Steps to Reproduce
+1.
+2.
+3.
+4.
+
+## Environment Details
```

Comment:
Let's ask for some more explicit details about the environment: node or browser version, sdk version, npm version.